src/Components/AdminIndex/AdminIndex.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+class AdminIndex extends React.Component {
+	constructor() {
+		super();
+		
+		this.state = {
+			inActivePeople: []
+		}
+	}
+
+	componentDidMount() {
+
+		fetch('http://localhost:3001/people/admin/list', {mode: "cors"})
+		.then(response => {
+			if (response.status >= 400) {
+		      throw new Error("Bad response from server");
+		    }
+		    return response.json();
+		})
+		.then(data => {
+			let inActivePeople = data[0].map(people => {
+				return(
+	  				<div className="col-md-2">
```

Comment:
Why store the jsx in state ??
Can we just have array of object and in render function we can have a map function that creates these jsx